### DISCUSSION TITLE
GPIO Strapping for expander flashing

### DISCUSSION THREAD

**Discussion by JensOgorek** [zauberzeug/lizard#60] [Ideas]
If an expander is to be flashed, the expander should check if flashing is possible using GPIO Strapping. If not, it should block the flashing attempt and notify why it was blocked.

**Comment by JensOgorek** [comment]
Moved to a more complex version on #69 that will work with IDF 5.3.1

**Comment by falkoschindler** [comment]
This feature has been released in version 0.5.0. ✔️
